### ISSUE TITLE
sys-process/psmisc fails to compile, but gets merged anyway

### ISSUE THREAD

**Issue by stefson** [gentoo/musl#214]
the most confusing thing about this is, the package gets merged despite emake failed for peekfd.c:  
```
make -j4 -C src peekfd 
make: Entering directory '/var/tmp/portage/sys-process/psmisc-23.1-r1/work/psmisc-23.1/src'
cc -O2 -pipe -march=armv7-a -mfpu=vfpv3-d16 -mfloat-abi=hard  -Wl,-O1 -Wl,--as-needed  peekfd.c   -o peekfd
peekfd.c: In function ‘print_version’:
peekfd.c:115:46: error: ‘VERSION’ undeclared (first use in this function); did you mean ‘__VERSION__’?
   fprintf(stderr, _("peekfd (PSmisc) %s\n"), VERSION);
                                              ^~~~~~~
                                              __VERSION__
peekfd.c:115:46: note: each undeclared identifier is reported only once for each function it appears in
peekfd.c: In function ‘main’:
peekfd.c:253:28: error: storage size of ‘regs’ isn’t known
    struct user_regs_struct regs;
                            ^~~~
make: *** [<builtin>: peekfd] Error 1
make: Leaving directory '/var/tmp/portage/sys-process/psmisc-23.1-r1/work/psmisc-23.1/src'
emake failed
```
full build log: [psmisc.zip](https://github.com/gentoo/musl/files/2800353/psmisc.zip)

**Comment by stefson** [comment]
possible reason: https://bugs.gentoo.org/330631

**Comment by stefson** [comment]
fixed in #215
